cpp/src/gandiva/precompiled/decimal_ops_test.cc

```diff
@@ -180,7 +180,7 @@ void TestDecimalSql::VerifyAllSign(DecimalTypeUtil::Op op, const DecimalScalar12
     // both -ve
     Verify(op, -left, -right, -expected_output, expected_overflow);
   } else {
-    DCHECK(op == DecimalTypeUtil::kOpMultiply || op == DecimalTypeUtil::kOpDivide);
+    ASSERT_TRUE(op == DecimalTypeUtil::kOpMultiply || op == DecimalTypeUtil::kOpDivide);
```

Comment:
```suggestion
    ASSERT_THAT(op, AnyOf(Eq(DecimalTypeUtil::kOpMultiply), Eq(DecimalTypeUtil::kOpDivide));
```
with:
```
using ::testing:Eq;
using ::testing::AnyOf;
```

Reply:
```
error: no member named 'AnyOf' in namespace 'testing'
```
I don't really want to bother, this PR is not about this :-)